app/views/trends/_extraction_warning.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if User.current.admin? %>
+  <%= alert :class => 'alert-warning', :header => _("This funcionality is deprecated"),
+            :text => ("and will be removed in version 2.2. "\
+                       "If you wish continue using it, please install a plugin Foreman Statistics. "\
+                       "For instructions follow " +
+                       link_to('Statistics Plugin manual', external_link_path(type: 'plugin_manual', name: 'foreman_statistics', version: '1.0'))
```

Comment:
I don't think `link_to` will work nicely in a translation string. I'd use ``%s` and interpolate

Reply:
I've not translated the string, as it will be up for most one version - no translation will get it in fast enough.